posthog/temporal/workflows/redshift_batch_export.py

```diff
@@ -69,17 +70,24 @@ def insert_records_to_redshift(
         )
         template = sql.SQL("({})").format(sql.SQL(", ").join(map(sql.Placeholder, columns)))
 
+        def flush_to_redshift():
+            psycopg2.extras.execute_values(cursor, query, batch, template)
+            ROWS_EXPORTED.labels(destination="redshift").inc(len(batch))
+            # It would be nice to record BYTES_EXPORTED for Redshift, but it's not worth estimating
+            # the byte size of each batch the way things are currently written. We can revisit this
+            # in the future if we decide it's useful enough.
```

Comment:
Yeah, Redshift is kind of limited as it doesn't support COPY-ing from local files, only from S3 😭. If/when we add COPY from S3 we can start tracking bytes too.